Sources/NIO/BaseSocket.swift

```diff
@@ -14,6 +14,49 @@
 
 import NIOConcurrencyHelpers
 
+#if os(Windows)
+import let WinSDK.EAFNOSUPPORT
+import let WinSDK.EBADF
+import let WinSDK.ENOENT
+
+import let WinSDK.FILE_ATTRIBUTE_REPARSE_POINT
+import let WinSDK.FILE_FLAG_BACKUP_SEMANTICS
+import let WinSDK.FILE_FLAG_OPEN_REPARSE_POINT
+import let WinSDK.FILE_SHARE_DELETE
+import let WinSDK.FILE_SHARE_READ
+import let WinSDK.FILE_SHARE_WRITE
+import let WinSDK.FileDispositionInfoEx
+
+import let WinSDK.GENERIC_READ
+
+import let WinSDK.INET_ADDRSTRLEN
+import let WinSDK.INET6_ADDRSTRLEN
+
+import let WinSDK.INVALID_HANDLE_VALUE
+import let WinSDK.INVALID_SOCKET
+
+import let WinSDK.IO_REPARSE_TAG_AF_UNIX
+
+import let WinSDK.NO_ERROR
+
+import let WinSDK.OPEN_EXISTING
+
+import func WinSDK.CloseHandle
+import func WinSDK.CreateFileW
+import func WinSDK.DeviceIoControl
+import func WinSDK.GetFileInformationByHandle
+import func WinSDK.GetFileType
+import func WinSDK.GetLastError
+import func WinSDK.SetFileInformationByHandle
+
+import struct WinSDK.BY_HANDLE_FILE_INFORMATION
+import struct WinSDK.DWORD
+import struct WinSDK.FILE_DISPOSITION_INFO
+import struct WinSDK.socklen_t
+
+import CNIOWindows
+#endif
+
 /// The requested UDS path exists and has wrong type (not a socket).
 public struct UnixDomainSocketPathWrongType: Error {}
```

Comment:
Can we move this error to `BSDSocketAPI.swift`? That'll fix the compile error in CI.

Reply:
Sure, but I dont understand why this change exposes the issue which should have existed previously.

Reply:
It's an infrastructural issue with the way we build our syscall investigation helpers, so the refactor exposed it. I wouldn't worry about it too much. :wink: